### DISCUSSION TITLE
Separate repo for Websockets (graphql-transport-ws) or add implementation to this one?

### DISCUSSION THREAD

**Discussion by valbers** [fsprojects/FSharp.Data.GraphQL#428] [Ideas]
Hi all,

I was thinking about publishing this as a separate repository and NuGet package: https://github.com/valbers/graphql-transport-ws.fsharp/tree/dev but then I thought: "you know what, what if I just added it to the existing FSharp.Data.GraphQL repo?" So I figured I'd ask you your opinion. What do you think?

**Comment by njlr** [comment]
My suggestion would be to put it in this repo so that we can easily release new versions together - however I'm not a maintainer!

A good example of this approach working well is [FsToolkit.ErrorHandling](https://github.com/demystifyfp/FsToolkit.ErrorHandling/tree/master/src).

**Comment by xperiandri** [comment]
I approve!
Please add both transport and samples.
